cmd/jobsResume.go

```diff
@@ -291,9 +289,8 @@ func (rca resumeCmdArgs) process() error {
 		// Get token from env var or cache.
 		if tokenInfo, err := uotm.GetTokenInfo(ctx); err != nil {
 			return err
-		} else {
-			credentialInfo.OAuthTokenInfo = *tokenInfo
 		}
+		credentialInfo.OAuthTokenInfo = *tokenInfo
```

Comment:
This one right here is a syntax error. `tokenInfo` is a local variable whose scope resides in the `if` statement above. it cannot be used outside `if`

Reply:
Hey @Simon, 

You can run `go build` & `go build ./e2etest` to check for syntax errors.